scripts/manifest.arm64.yml

```diff
@@ -0,0 +1,32 @@
+---
+version: 1.0.0.0-rc1
+
+products:
+  opensearch: 
+    opensearch-min: https://artifacts.opensearch.org/snapshots/core/opensearch/1.0.0/opensearch-min-1.0.0-SNAPSHOT-linux-arm64-189.tar.gz
+    libs:
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-knnlib-1.0.0.0-rc1-linux-arm64.zip
+    plugins:
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-sql-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-alerting-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-job-scheduler-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-security-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-performance-analyzer-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-index-management-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-knn-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-anomaly-detection-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-reports-scheduler-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-asynchronous-search-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/opensearch_plugins/opensearch-notebooks-1.0.0.0-rc1.zip
+  opensearch-dashboards: 
+    opensearch-dashboards-min: https://artifacts.opensearch.org/snapshots/core/opensearch-dashboards/1.0.0/opensearch-dashboards-min-1.0.0-SNAPSHOT-linux-arm64-110.tar.gz
+    plugins:
+        - s3://staging.opensearch.amazon.com/plugins-temp/dashboard_plugins/queryWorkbenchDashboards-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/dashboard_plugins/anomalyDetectionDashboards-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/dashboard_plugins/securityDashboards-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/dashboard_plugins/alertingDashboards-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/dashboard_plugins/indexManagementDashboards-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/dashboard_plugins/notebooksDashboards-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/dashboard_plugins/ganttChartDashboards-1.0.0.0-rc1.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/dashboard_plugins/reportsDashboards-1.0.0.0-rc1-linux-arm64.zip
+        - s3://staging.opensearch.amazon.com/plugins-temp/dashboard_plugins/traceAnalyticsDashboards-1.0.0.0-rc1.zip
```

Comment:
Consider adding a `schema-version` field to the manifest so we can evolve the format.

Reply:
Done